src/encoded/static/scss/encoded/modules/_matrix.scss

```diff
@@ -15,6 +18,390 @@ $row-data-header-width: 200px;
+            }
+        }
+        .body-inset-container {
+            flex: 0 1 60%;
+            text-align: center;
+            .body-inset {
+                img {
+                    width: 55px;
+                    height: 55px;
+                }
+            }
+            .body-inset:after {
+                width: 55px;
+                height: 55px;
+            }
+        }
+        .body-list-narrow {
+            flex: 0 1 40%;
+        }
+    }
+    .body-facet :nth-child(1) {
+        order: 1;
+    }
+    .body-facet :nth-child(2) {
+        order: 2;
+    }
+    .body-facet :nth-child(3) {
+        order: 4;
+    }
+    .body-facet :nth-child(4) {
+        order: 3;
+    }
+    .clickable-diagram-container {
+        .body-list-inner {
+            .body-list-element {
+                width: auto;
+            }
+        }
+    }
+}
+
+.expand-arrows, .collapse-arrows {
+    height: 30px;
+    width: 30px;
+    position: absolute;
+    top: 43px;
+    left: 5px;
+}
+.expand-arrows {
+    fill: $Homo-sapiens;
+    @media screen and (max-width: $screen-sm-min) {
+        top: 8px;
+        fill: white;
+        height: 20px;
+        width: 20px;
+    }
+    @media screen and (max-width: $screen-xs-min) {
+        top: 3px;
+    }
+}
+.collapse-arrows {
+    fill: white;
+    top: 2px;
+    z-index: 999;
+    @media screen and (max-width: $screen-sm-min) {
+        top: 8px;
+        height: 20px;
+        width: 20px;
+    }
+    @media screen and (max-width: $screen-xs-min) {
+        top: 3px;
+    }
+}
+.body-map-expander, .body-map-collapser {
+    background: $Homo-sapiens;
+    text-align: center;
+    font-family: Mada;
+    padding: 10px;
+    color: white;
+    font-weight: 400;
+    cursor: pointer;
+    position: relative;
+    &:hover {
+        background: darken($Homo-sapiens, 10%);
+        font-weight: 600;
+    }
+    @media screen and (max-width: $screen-sm-min) {
+        font-size: 1rem;
+    }
+    @media screen and (max-width: $screen-xs-min) {
+        padding: 5px;
+        font-size: 0.8rem;
+    }
+}
+
+.ReferenceEpigenomeMatrix {
+    .matrix-header__title {
+        padding-bottom: 5px;
+    }
+    .test-project-selector {
+        label {
+            margin-left: 5px;
+        }
+    }
+    .matrix-header {
+        border-bottom: 0;
+    }
+    .header-clear-links {
+        margin-top: 20px;
+        @media screen and (max-width: $screen-sm-min) {
+            margin: 10px 5px;
+        }
+        button {
+            position: relative;
+            top: 0;
+            bottom: 0;
+            right: 0;
+            left: 0;
+            @media screen and (max-width: $screen-sm-min) {
+                width: auto;
+            }
+        }
+    }
+    .tab-nav-Homosapiens {
+        border-bottom: 2px solid $Homo-sapiens;
+    }
+    .tab-nav-Musmusculus {
+        border-bottom: 2px solid $Mus-musculus;
+    }
+    .nav-tabs {
+        li {
+            margin-bottom: -1px;
```

Comment:
Negative margin seem like a code smell. Is there a reason an alternative is not sufficient?

Reply:
I increased the border width and therefore added the negative margin to not have an extra line at the bottom of the tab.